src/column.rs

```diff
@@ -137,7 +142,11 @@ pub fn hash_key(key: &[u8], salt: &Salt, uniform: bool, db_version: u32) -> Key
 			}
 		}
 	} else {
-		k.copy_from_slice(blake2_rfc::blake2b::blake2b(32, salt, key).as_bytes());
+		let mut ctx = Blake2bMac::<U32>::new_with_salt_and_personal(salt, &[], &[])
+			.expect("Salt length (32) is a valid key length (<= 64)");
```

Comment:
Here the naming is a bit misleading.

That is, with previous implementation the `salt` was used as the `blake2b-rfc` "key" argument while the `key` was used as the "data" argument.

The `new_with_salt_and_personal` has a dedicated parameter (the 2nd) for the salt. But to obtain the same MAC as before we have to pass `salt` as the 1st parameter